hugegraph-hbase/src/main/java/com/baidu/hugegraph/backend/store/hbase/HbaseSessions.java

```diff
@@ -712,7 +712,11 @@ public RowIterator scan(String table,
                                 byte[] stopRow, boolean inclusiveStop) {
             assert !this.hasChanges();
 
-            Scan scan = new Scan().withStartRow(startRow, inclusiveStart);
+            Scan scan = new Scan();
+            if (startRow != null) {
```

Comment:
could you also update `HbaseTable.HbaseShardSpliter.position()`:
```java
        public byte[] position(String position) {
            if (START.equals(position) || END.equals(position)) {
                return null;
            }
            return StringEncoding.decodeBase64(position);
        }
```

Reply:
done